src/charm.py

```diff
@@ -555,6 +555,8 @@ def enable_disable_extensions(self, database: str = None) -> None:
             enable = self.config[plugin]
             # Enable or disable the plugin/extension.
             extension = "_".join(plugin.split("_")[1:-1])
+            if extension == "uuid_ossp":
+                extension = '"uuid-ossp"'
```

Comment:
Might be better to add an override mapping so in the future we can add other exceptions there.